flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/sort/KeyAndValueSerializer.java

```diff
@@ -41,7 +41,7 @@
  *      [key-length] | &lt;key&gt; | &lt;timestamp&gt; | &lt;record&gt;
  * </pre>
  */
-final class KeyAndValueSerializer<IN> extends TypeSerializer<Tuple2<byte[], StreamRecord<IN>>> {
+public final class KeyAndValueSerializer<IN> extends TypeSerializer<Tuple2<byte[], StreamRecord<IN>>> {
```

Comment:
We shouldn't make this `public` just to make it accessible in the test. We can just use a `"String"` for exclusion.